Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -8,29 +8,30 @@
 import AnyCodable
 import Foundation
 
+public protocol MetricNameProtocol {
+    var value: String { get }
+}
+
 public struct Event: Equatable {
-    public enum EventName: Equatable {
-        case OpenedPush
-        case ViewedProduct
-        case SearchedProducts
-        case StartedCheckout
-        case PlacedOrder
-        case OrderedProduct
-        case CancelledOrder
-        case PaidForOrder
-        case SubscribedToBackInStock
-        case SubscribedToComingSoon
-        case SubscribedToList
-        case SuccessfulPayment
-        case FailedPayment
-        case CustomEvent(String)
+    public struct Legacy: Equatable {
+        public enum MetricName: MetricNameProtocol {
+            case OpenedPush
+            case ViewedProduct
+            case StartedCheckout
+            case ActiveOnSite
+            case AddedToCart
```

Comment:
Are these the only non-custom events we now support? Asking since on RN I've added the old list.

Reply:
Yah we updated it we should make RN match this.

Reply:
Uhh also opened app instead of active of site.